docs/config.rst

```diff
@@ -13,46 +13,89 @@ Defining sessions
 .. autofunction:: nox.session
 
 Nox sessions are configured via standard Python functions that are decorated
-with ``@nox.session`` or start with ``session_``. For example, these are all
-sessions::
+with ``@nox.session``. For example::
 
     import nox
 
-    def session_a(session):
+    @nox.session
+    def b(session):
         pass
 
-    def session_123(session):
+You can also configure sessions to run against multiple Python versions as described in :ref:`virtualenv config` and  parametrize sessions as described in :ref:`parametrized sessions <parametrized>`.
+
+.. _virtualenv config:
+
+Configuring a session's virtualenv
+----------------------------------
+
+By default, Nox will create a new virtualenv for each session using the same interpreter that Nox uses. If you installed Nox using Python 3.6, Nox will use Python 3.6 by default for all of your sessions.
+
+You can tell Nox to use a different Python interpreter/version by specifying the ``python`` argument to ``@nox.session``::
+
+    @nox.session(python='2.7')
+    def tests(session):
         pass
 
-    @nox.session
-    def b(session):
+You can also tell Nox to run your session against multiple Python interpreters. Nox will create a separate virtualenv and run the session for each interpreter you specify. For example, this session will run twice - once for Python 2.7 and once for Python 3.6::
+
+    @nox.session(python=['2.7', '3.6'])
+    def tests(session):
         pass
 
-These are **not**::
+You can use ``nox.python`` to specify even more advanced configuration. For example, if you want to turn off the virtualenv creation::
 
-    def some_session(session):
+    @nox.session(python=nox.python(virtualenv=False))
+    def tests(session):
         pass
 
-    def other_func(session):
+You can also specify that the virtualenv should *always* be reused or change the virtualenv name::
+
+    @nox.session(
+        python=nox.python(
+            python='3.6',
+            virtualenv='custom-name',
+            reuse=True))
+    def tests(session):
         pass
 
-You may define sessions using either the decorator or the naming convention.
-This can affect the execution order as described in the
-:ref:`usage docs<session_execution_order>`.
+Finally, you can use multiple instances of ``nox.python`` to specify more than one customization::
+
+    @nox.session(
+        python=[
+            nox.python(
+                python='3.6',
+                virtualenv='custom-name',
+                reuse=True),
+            nox.python(
+                python='3.5',
+                reuse=True),
+        ])
+    def tests(session):
+        pass
+
+When collecting your sessions, Nox will create a separate session for each interpreter. You can see these sesions when running ``nox --list-sessions``. For example this Noxfile::
+
+    @nox.session(python=['2.7', '3.5', 3.6', '3.7'])
+    def tests(session):
+        pass
+
+Will produce these sessions::
```

Comment:
If there is only one interpreter, is the suffix applied? (I think it should be.)